src/test/java/org/prebid/server/auction/ExchangeServiceTest.java

```diff
@@ -198,11 +199,30 @@ public void shouldTolerateImpWithUnknownBidderInExtension() {
 
         // then
         verify(bidderCatalog).isValidName(eq("invalid"));
-        verifyNoMoreInteractions(bidderCatalog);
+        verify(bidderCatalog).isDeprecatedName(eq("invalid"));
         verifyZeroInteractions(bidderRequester);
         assertThat(bidResponse).isNotNull();
     }
 
+    @Test
+    public void shouldProccesRequestAndAddErrorAboutDeprecatedBidder() {
+        String invalidBidderName = "invalid";
+
+        given(bidderCatalog.isValidName(invalidBidderName)).willReturn(false);
```

Comment:
Please divide parts of tests in given/when/then blocks with comments just as it is done in other tests

Reply:
fixed, dunno why github don`t sees it